com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/FontBuilder.java

```diff
@@ -23,19 +23,17 @@
 import com.dynamo.bob.Task;
 import com.dynamo.bob.font.Fontc;
 import com.dynamo.bob.fs.IResource;
-import com.dynamo.bob.util.BobProjectProperties;
 
 import com.dynamo.render.proto.Font.FontDesc;
 import com.dynamo.render.proto.Font.FontMap;
 import com.dynamo.render.proto.Font.FontTextureFormat;
 
-@ProtoParams(srcClass = FontDesc.class, messageClass = FontDesc.class)
-@BuilderParams(name = "Font", inExts = ".font", outExt = ".fontc")
+@ProtoParams(srcClass = FontDesc.class, messageClass = FontMap.class)
```

Comment:
This is the fix of the initial issue. Our output format is `FontMap` not `FontDesc`